docs/user/dashboard/dashboard.asciidoc

```diff
@@ -197,8 +197,18 @@ With *Lens*, you can:
 * Create visualizations with multiple layers and indices.
 * Change the aggregation and labels to customize the data.
 
-[role="screenshot"]
-image::images/lens_drag_drop.gif[Drag and drop]
+++++
+<script type="text/javascript" async src="https://play.vidyard.com/embed/v4.js"></script>
+<img
```

Comment:
This is a great idea! However, this is a quick start for logging. Is there a quick start specific to Lens?

Also, it would be nice to have a border around the video, something like role=screenshot provides.

Reply:
According to the AsciiDoc User Guide, we can control the behavior of the videos,  but not the look.